config.py

```diff
@@ -33,6 +34,10 @@ def __init__(self):
         self.create_xray_when_sending.setChecked(prefs['create_xray_when_sending'])
         self.l.addWidget(self.create_xray_when_sending)
 
+        self.expand_aliases = QCheckBox('Auto associate split aliases')
```

Comment:
Yeah I sat there thinking about what to name this. I asked my boyfriend if he understood what "Auto-expand aliases" means and he said no and we came up with "Auto associate split aliases" -- Auto-expand aliases sounds like we're not expanding anything at all -- e.g. "timothy cratchit" would not be split into "timothy" and "cratchit", where in reality we will be, we just won't be turning "Tiny Tim" into "tiny" and "tim".. i forgot to update the readme.. thanks for the reminder